test/e2e/edu-1990-rjsf-helpers.js

```diff
@@ -0,0 +1,90 @@
+import _ from 'lodash/fp';
+
+function completeBenefitRelinquishment(client, data) {
+  client
+    .selectRadio('root_view:benefitsRelinquishedContainer_benefitsRelinquished', data['view:benefitsRelinquishedContainer'].benefitsRelinquished)
+    .fillDate('root_view:benefitsRelinquishedContainer_benefitsRelinquishedDate', data['view:benefitsRelinquishedContainer'].benefitsRelinquishedDate);
+}
+
+function completeBenefitsSelection(client, data) {
+  const benefits = data['view:selectedBenefits'];
+  client
+    .clickIf('input[name="root_view:selectedBenefits_chapter33"]', benefits.chapter33)
+    .clickIf('input[name="root_view:selectedBenefits_chapter30"]', benefits.chapter30)
+    .clickIf('input[name="root_view:selectedBenefits_chapter1606"]', benefits.chapter1606)
+    .clickIf('input[name="root_view:selectedBenefits_chapter32"]', benefits.chapter32);
```

Comment:
Was `clickIf` causing IE sauce labs failures? Do Remember that right?

Reply:
That sounds vaguely true? I will run these in sauce and see.

Reply:
They run successfully on Sauce for IE.